src/cli/facade/dev.js

```diff
@@ -157,7 +157,7 @@ module.exports = function(dependencies) {
                 return callback(err);
               }
 
-              const registry = new oc.Registry({
+              const registry = oc.Registry({
```

Comment:
won't this change break how registry is initialised right now outside of this repository?

Reply:
why would it? it's not a class, so what will be the difference outside?

Reply:
We've tested it together and it does not seem to affect the initalisation after all.